presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -113,6 +113,18 @@ bool isCacheTtlEnabled() {
 
 } // namespace
 
+const char* nodeState2String(NodeState nodeState) {
+  switch (nodeState) {
+    case presto::NodeState::kActive:
+      return "active";
+    case presto::NodeState::kInActive:
+      return "inactive";
+    case presto::NodeState::kShuttingDown:
+      return "shutting_down";
```

Comment:
```
std::string nodeState2String() {

default:
   return fmt::format("<unknow:{}>", static_cast<int>(nodeState));
```

Reply:
Using 'default' is not a great idea: when new enum members are added you will not have compiler warning -> error for them not used in the switch statement.
That's why I prefer having a separate return. We can add the number though.